[2021-03-17 19:28:36,812] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: taxi.CsvExampleGen 2021-03-17T23:28:18.372751+00:00 [queued]>
[2021-03-17 19:28:36,823] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: taxi.CsvExampleGen 2021-03-17T23:28:18.372751+00:00 [queued]>
[2021-03-17 19:28:36,823] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-17 19:28:36,823] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-17 19:28:36,823] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-17 19:28:36,831] {taskinstance.py:1063} INFO - Executing <Task(AirflowComponent): CsvExampleGen> on 2021-03-17T23:28:18.372751+00:00
[2021-03-17 19:28:36,836] {standard_task_runner.py:52} INFO - Started process 29150 to run task
[2021-03-17 19:28:36,849] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'taxi', 'CsvExampleGen', '2021-03-17T23:28:18.372751+00:00', '--job-id', '17', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/taxi_pipeline.py', '--cfg-path', '/var/folders/j6/755w2l7n4x1c9313788klj880000gq/T/tmpljpzgi6j', '--error-file', '/var/folders/j6/755w2l7n4x1c9313788klj880000gq/T/tmpmwf4ur10']
[2021-03-17 19:28:36,852] {standard_task_runner.py:77} INFO - Job 17: Subtask CsvExampleGen
[2021-03-17 19:28:36,929] {logging_mixin.py:104} INFO - Running <TaskInstance: taxi.CsvExampleGen 2021-03-17T23:28:18.372751+00:00 [running]> on host 152.1.168.192.in-addr.arpa
[2021-03-17 19:28:36,972] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=taxi
AIRFLOW_CTX_TASK_ID=CsvExampleGen
AIRFLOW_CTX_EXECUTION_DATE=2021-03-17T23:28:18.372751+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-17T23:28:18.372751+00:00
[2021-03-17 19:28:36,984] {base_component_launcher.py:196} INFO - Running driver for CsvExampleGen
[2021-03-17 19:28:36,995] {metadata_store.py:93} INFO - MetadataStore with DB connection initialized
[2021-03-17 19:28:36,997] {utils.py:631} INFO - select span and version = (0, None)
[2021-03-17 19:28:36,998] {utils.py:640} INFO - latest span and version = (0, None)
[2021-03-17 19:28:37,048] {base_component_launcher.py:212} INFO - Running publisher for CsvExampleGen
[2021-03-17 19:28:37,051] {metadata_store.py:93} INFO - MetadataStore with DB connection initialized
[2021-03-17 19:28:37,054] {python.py:118} INFO - Done. Returned value was: None
[2021-03-17 19:28:37,063] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=taxi, task_id=CsvExampleGen, execution_date=20210317T232818, start_date=20210317T232836, end_date=20210317T232837
[2021-03-17 19:28:37,084] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-17 19:28:37,120] {local_task_job.py:146} INFO - Task exited with return code 0
